libraries/chain/include/eosio/chain/finality/finalizer.hpp

```diff
@@ -184,7 +189,13 @@ namespace eosio::chain {
 
       // for testing purposes only, not thread safe
       const fsi_t& get_fsi(const bls_public_key& k) const { return finalizers.at(k).fsi; }
-      void         set_fsi(const bls_public_key& k, const fsi_t& fsi) { finalizers[k].fsi = fsi; }
+
```

Comment:
I think remove this blank line or add another comment saying this is only for testing purposes and not thread safe.

Reply:
removed!